opentelemetry-sdk/CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ LogData {
 }
 ```
 The `LogRecord::target` field contains the actual target/component emitting the logs, while the `Instrumentation::name` contains the name of the OpenTelemetry appender.
+- Update to `http` v1 [#1674](https://github.com/open-telemetry/opentelemetry-rust/pull/1674)
```

Comment:
Does hyper upgrade impacts opentelemetry-sdk in anyway?

Reply:
The `HttpClient` trait appears in the `Sampler::jaeger_remote()` API.

Reply:
Updated the changelog note to be a bit more detailed